### PR TITLE
WIP: Helm migration

### PR DESCRIPTION
**Pull request by sabre1041**
Migration of provisioning and management from Kustomize to Helm

### CODE REVIEW

File: ansible/roles/redhat-image-downloader/defaults/main.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ redhat_sso_url: https://sso.redhat.com/auth/realms/redhat-external/protocol/open
 redhat_api_url: https://api.access.redhat.com/management/v1
 
 # image_checksums represents a list of SHA-256 hashes that correspond to specific
-# Red Hat images.  The RHEL 8.4 KVM QCOW2 Image followed by the RHEL 8.4 Boot ISO
+# Red Hat images.  The RHEL 8.5 KVM QCOW2 Image followed by the RHEL 8.5 Boot ISO
 # are defaults in this list.
 # 
 # The latest can be found here:
-# https://access.redhat.com/downloads/content/479/ver=/rhel---8/8.4/x86_64/product-software
+# https://access.redhat.com/downloads/content/479/ver=/rhel---8/8.5/x86_64/product-software
```
Comment (jordigilh) [inline ansible/roles/redhat-image-downloader/defaults/main.yaml:11]: I wonder if we should consider at this stage to support RHEL for ARM64 and extract the architecture from the charts. WDYT?

File: helm/testing/operators/sealed-secret-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
```
Comment (jordigilh) [inline helm/testing/operators/sealed-secret-operator.yaml:1]: I thought we decided not to use the sealed secret operator anymore. Are we keeping it for now and doing a cleanup post merge?

Comment (nasx) [reply helm/testing/operators/sealed-secret-operator.yaml:1]: This is going to get dropped (along w/ all of `helm/testing`) in my docs PR (#206).

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,9 @@
 local/
 openshift/gitops/clusters/overlays/byo/bootstrap/redhat-portal-credentials
```
[the remark this answers is not in the document]
Comment (nasx) [reply .gitignore:2]: Yeah good catch, can you submit a PR?

File: ansible/roles/redhat-image-downloader/defaults/main.yaml (hunk 1 above)
Comment (nasx) [reply ansible/roles/redhat-image-downloader/defaults/main.yaml:11]: Created two issues (#210 and #211) to track this.

**Review by nasx — APPROVED** [review]
LGTM!

**Pull request merged**